tests/integration/tier1/single-yum-transaction/test_check_for_latest_packages.py

```diff
@@ -0,0 +1,36 @@
+from conftest import SYSTEM_RELEASE_ENV
+from envparse import env
+
+
+PACKAGES_TO_VERIFY_RHEL_7 = [("shim-x64", "shim-x64-15.6-3.el7_9.x86_64")]
+# Match the latest version used for conversion, currently Oracle Linux 8.7
+PACKAGES_TO_VERIFY_RHEL_8_X = [("shim-x64", "shim-x64-15.6-1.el8.x86_64")]
+# Mainly for CentOS Linux 8.5
+PACKAGES_TO_VERIFY_RHEL_8_5 = [("shim-x64", "shim-x64-15.4-2.el8_1.x86_64")]
+
+
+def test_packages_upgraded_after_conversion(convert2rhel, shell):
```

Comment:
Please add the marker, also add to `pytest.ini`, please.
```suggestion

@pytest.mark.test_packages_upgraded_after_conversion
def test_packages_upgraded_after_conversion(convert2rhel, shell):
```